Sources/DiagnosticsReport.swift

```diff
@@ -30,7 +30,18 @@ public extension DiagnosticsReport {
         let simulatorPath = (NSSearchPathForDirectoriesInDomains(.desktopDirectory, .userDomainMask, true) as [String]).first!
         let simulatorPathComponents = URL(string: simulatorPath)!.pathComponents.prefix(3).filter { $0 != "/" }
         let userPath = simulatorPathComponents.joined(separator: "/")
-        let path = "/\(userPath)/Desktop/Diagnostics/\(filename)"
+        let path = "/\(userPath)/Desktop"
+        let directoryPath = "\(path)/Diagnostics"
+        do {
+            try FileManager.default.createDirectory(atPath: directoryPath, withIntermediateDirectories: false, attributes: nil)
+            save(path: "\(directoryPath)/\(filename)")
+        } catch {
+            save(path: "\(path)/\(filename)")
+        }
```

Comment:
Can we instead make this more explicit using https://stackoverflow.com/a/24696209/1485701 for example?

So we actually check if the directory exists, otherwise, we create it.

Also, isn't the `createDirectory` method failing if the directory already exists?